sdk/identity/azure-identity/src/main/java/com/azure/identity/KeyringItemSchema.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+/**
+ * An expandable enum for types of item schema in a Keyring.
+ */
+public final class KeyringItemSchema {
```

Comment:
Since the javadoc mentions that this is an expandable enum, should this class extend from `ExpandableStringEnum`?

Reply:
Changed to extend from `ExpandableStringEnum`